packages/Button/src/Button.js

```diff
@@ -4,12 +4,10 @@ import "what-input";
 import RawButton from "@paprika/raw-button";
 import RefreshIcon from "@paprika/icon/lib/Refresh";
 import DownIcon from "@paprika/icon/lib/CaretDown";
-import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
```

Comment:
Two things on top of my mind:
1.- enums / types / constants should be their own package.
2.- ShirtSize and ButtonKinds are not descriptive and when use it on prototypes documentation react docgen sometimes is not capable to list exactly what to write on it, also are not memorable neither easier to remember in specific situations.

My proposal:

As a user I would like to see something like this:
```js
import { button } from "@paprika/constants";

<Button kind={button.kind.MEDIUM} />
```

```js
import { datagrid } from "@paprika/constants";

<DataGrid borderTypes={datagrid.borderType.HORIZONTAL} />
```

**Namespace** format for the constant: `component` / ` prop`  / `constant value`

Doing this is easier to know where to look for the value you don't have to imagine or search on the code to figure out the value even is easier to do:

```js
import { listBox } from "@paprika/constants";
console.log(listbox.size)
```

To see if what are the value available or even just log listBox and see all the available constants.


With this approach, I think our constants are flexible enough to escalate properly and be easy to remember.